config-linux.md

```diff
@@ -521,6 +521,30 @@ Its value is either slave, private, or shared.
     "rootfsPropagation": "slave",
 ```
 
+## Masked Paths
+
+maskedPaths will mask over the provided paths inside the container so that they cannot be read.
+
+###### Example
+
+```json
+    "maskedPaths": [
+        "/proc/kcore"
+    ]
+```
+
+## Readonly Paths
+
+readonlyPaths will set the provieded paths as readonly inside the container.
```

Comment:
“provieded” → “provided”.  And probably **`readonlyPaths`** (and similar for your earlier **`maskedPaths`** docs) to match the rest of the spec.